### DISCUSSION TITLE
Marker bond to only one model

### DISCUSSION THREAD

**Discussion by gavransava** [patrykandpatrick/vico#928] [Questions] [answered]
### Question

I have a line chart with 2 or 3 models. Is there a way to setup a marker to only work for one specific model, to be displayed; the label and the oval marker. Thanks 

### Vico version(s)

1.15.0

**Comment by Gowsky** [comment, marked as answer]
Hello, @gavransava. Yes, you can do this by overriding the `draw` function. You can filter `markedEntries` so that only the `Marker.EntryModel` for the desired series is preserved. Then, you can call `super.draw` with the new list.

**Reply by gavransava** [reply]
Thanks :)
